docs/uz/docs/index.md

```diff
@@ -0,0 +1,45 @@
+<p align="center">
```

Comment:
It looks like there's a few lines missing from the original file. Have a look at [`en/docs/index.md`](https://github.com/fastapi/fastapi/blob/master/docs/en/docs/index.md?plain=1) , which starts with a header and some `<style>` tags - these should be included here (without change).